docs/docs/api/Tasks/fetchSingleTask.md

```diff
@@ -7,68 +7,68 @@ borderRadius: '2px',
 color: '#E83E8C',
 }}>{children}</span> );
 
-<Endpoint>GET tasks/task/:id/</Endpoint>: Fetching a single Task
+<Endpoint>GET /tasks/task/:id</Endpoint>: Fetching a single Task
 
 ```json
-{
-
-}
+{}
 ```
 
 ### Example Request
+
 This is a **protected route**, a **valid JWT is required** in the header field
+
 #### Header
+
 ```
 Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
 ```
 
 #### Body
-```json
-{
 
-}
+```json
+{}
 ```
 
 #### Parameters
-```json
-{
-    id: task_id // Id of the task that is being searched for
-}
 
-```
+    id: task_id // Id of the task that is being searched for
```

Comment:
We can remove the need for the comment where it says `// Id of the task that is being searched for`.
`task_id` already conveys that information to us without the need to include the comment